.github/workflows/test.yml

```diff
@@ -0,0 +1,39 @@
+on:
+  # Trigger the workflow on push or pull request, but only for the develop branch
+  push:
+    branches:
+      - develop
+  pull_request:
+    branches:
+      - develop
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [ '3.8', '3.9', '3.10-dev' ]
+
+    name: Python ${{ matrix.python-version }} on Ubuntu latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+          architecture: x64
+      - name: Install system dependencies
+        run: |
+          sudo apt-get update -qq
+          sudo apt-get install -qq -y gir1.2-gtk-3.0 gobject-introspection libgirepository1.0-dev xvfb
+      - name: Install python dependencies
+        run: |
+          pip install PyGObject
+          pip install pyxdg
+      - name: Configure git
+        run: |
+          git config --global user.email "you@example.com"
+          git config --global user.name "Your Name"
+      - name: Test
+        run: LC_COLLATE="en_US.UTF-8" xvfb-run ./test.py
```

Comment:
`LC_COLLATE="en_US.UTF-8"` is required to fix the following error:

```
======================================================================
FAIL: runTest (tests.widgets.TestLinkEntry)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/zim-desktop-wiki/zim-desktop-wiki/tests/widgets.py", line 254, in runTest
    TestPageEntry.runTest(self)
  File "/home/runner/work/zim-desktop-wiki/zim-desktop-wiki/tests/widgets.py", line 229, in runTest
    self.assertEqual([r[0] for r in self.completion_model], wanted)
AssertionError: Lists differ: [':žžž', ':Bar', ':Maßnahmen', ':Placeholder', ':Test'] != [':Bar', ':Maßnahmen', ':Placeholder', ':Test', ':žžž']

First differing element 0:
':žžž'
':Bar'

- [':žžž', ':Bar', ':Maßnahmen', ':Placeholder', ':Test']
?  --------

+ [':Bar', ':Maßnahmen', ':Placeholder', ':Test', ':žžž']
?                                               ++++++++


======================================================================
FAIL: runTest (tests.widgets.TestNamespaceEntry)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/zim-desktop-wiki/zim-desktop-wiki/tests/widgets.py", line 245, in runTest
    TestPageEntry.runTest(self)
  File "/home/runner/work/zim-desktop-wiki/zim-desktop-wiki/tests/widgets.py", line 229, in runTest
    self.assertEqual([r[0] for r in self.completion_model], wanted)
AssertionError: Lists differ: [':žžž', ':Bar', ':Maßnahmen', ':Placeholder', ':Test'] != [':Bar', ':Maßnahmen', ':Placeholder', ':Test', ':žžž']

First differing element 0:
':žžž'
':Bar'

- [':žžž', ':Bar', ':Maßnahmen', ':Placeholder', ':Test']
?  --------

+ [':Bar', ':Maßnahmen', ':Placeholder', ':Test', ':žžž']
?                                               ++++++++


======================================================================
FAIL: runTest (tests.widgets.TestPageEntry)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/zim-desktop-wiki/zim-desktop-wiki/tests/widgets.py", line 229, in runTest
    self.assertEqual([r[0] for r in self.completion_model], wanted)
AssertionError: Lists differ: [':žžž', ':Bar', ':Maßnahmen', ':Placeholder', ':Test'] != [':Bar', ':Maßnahmen', ':Placeholder', ':Test', ':žžž']

First differing element 0:
':žžž'
':Bar'

- [':žžž', ':Bar', ':Maßnahmen', ':Placeholder', ':Test']
?  --------

+ [':Bar', ':Maßnahmen', ':Placeholder', ':Test', ':žžž']
?
```

See https://github.com/pdecat/zim-desktop-wiki/pull/1/checks?check_run_id=3708067156#step:7:995